tests/test_nginx_role.py

```diff
@@ -0,0 +1,32 @@
+from tests.helm_template_generator import render_chart
+
+
+class TestNginxRole:
+    def test_nginx_role(self):
+        # sourcery skip: extract-duplicate-method
+        docs = render_chart(
+            show_only=["charts/nginx/templates/nginx-role.yaml"],
+        )
+
+        assert len(docs) == 1
+
+        doc = docs[0]
+
+        assert doc["kind"] == "Role"
+        assert doc["apiVersion"] == "rbac.authorization.k8s.io/v1"
+        assert doc["metadata"]["name"] == "RELEASE-NAME-nginx"
+
+    def test_nginx_role_verbs_ingresses_status(self):
+        # sourcery skip: extract-duplicate-method
+        docs = render_chart(
+            values={"global": {"singleNamespace": "true"}},
+            show_only=["charts/nginx/templates/nginx-role.yaml"],
+        )
+
+        assert len(docs) == 1
+        doc = docs[0]
+
+        assert doc["kind"] == "Role"
+        assert doc["apiVersion"] == "rbac.authorization.k8s.io/v1"
+        assert doc["metadata"]["name"] == "RELEASE-NAME-nginx"
+        assert doc["rules"] == any("ingresses/status" in val for val in doc.values())
```

Comment:
This is failing because as it is written, `doc["rules"]` would be of type `boolean`, which will never be true. Try running this through `venv/bin/python -m pytest -sv --pdb tests -k 'nginx_role'` and see if you can fix the assertion you had in mind. If you can't come up with anything I have some ideas on how we could do this. If nothing else we could just assert against the entire value of `doc["rules"]`, though I'm not sure that's the most elegant test.